x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/http/sender/ChatCompletionInput.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.inference.external.http.sender;
+
+import java.util.List;
+import java.util.Objects;
+
+public class ChatCompletionInput extends InferenceInputs {
```

Comment:
I feel like we should somehow indicate that this is different from the UnifiedChatInput. Perhaps a javadoc comment mentioning that, or adding the word "legacy" to the class name?

Reply:
Yeah good point, I'll add some.

Reply:
thank you!